src/Bridges/Objective/map.jl

```diff
@@ -0,0 +1,81 @@
+"""
+    Map <: AbstractDict{MOI.ObjectiveFunction, AbstractBridge}
+
+Mapping between bridged variables and the bridge that bridged the variable.
+"""
+mutable struct Map <: AbstractDict{MOI.ObjectiveFunction, AbstractBridge}
```

Comment:
Does using `AbstractBridge` instead of `Any` hurt/help performance? (There were some recent discussions on Discourse about bad compile times from abstract types in dicts, e.g., https://discourse.julialang.org/t/compiler-performance-of-dict/27986/12?u=odow)

Reply:
I didn't know that it increase compile time, it's there mostly for type-safety, to make sure to get an error if anything else is stored there.
We can check if changing this improves things but I'd say it's out of the scope of this PR.